pkg/kubelet/cm/container_manager.go

```diff
@@ -154,6 +154,12 @@ type ContainerManager interface {
 	// Updates returns a channel that receives an Update when the device changed its status.
 	Updates() <-chan resourceupdates.Update
 
+	// PodHasExclusiveCPUs returns true if the provided pod has containers with exclusive cpu
```

Comment:
nit: if you resubmit, let's add rephrase as
```
PodHasExclusiveCPUs returns true if the provided pod has containers with exclusive CPUs,
this means that at least one sidecar container or one app container has exclusive CPUs allocated.
```

Reply:
resolved!